README.md

```diff
@@ -2,3 +2,16 @@
 [![Build Status](https://travis-ci.org/SMaxOwok/ui-pictionary.svg?branch=master)](https://travis-ci.org/SMaxOwok/ui-pictionary)
 
 ![picationary](static/pictionary.gif)
+
+# Setup
+Requirements: Ruby version 2.5.7, Bundler 2
+- Install nvm and yarn
+  - You may need to `yarn cache clean` and then `yarn install`
+- The local webpack server is run with `ruby bin/webpack-dev-server`
```

Comment:
May need to go a little further with these...  Since there are so many processes, I think it's best run with `foreman`, which they'll need globally.